Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/api/ui/TestForAPIFragment.kt

```diff
@@ -573,6 +586,10 @@ class TestForAPIFragment : Fragment(R.layout.fragment_test_for_a_p_i),
         toast.show()
     }
 
+    private fun showSnackBar(message: String) {
+        view?.let { Snackbar.make(it, message, Snackbar.LENGTH_LONG) }?.show()
```

Comment:
? before show() actually needed?

Reply:
message as string is nullable. hence the check. I can just check if String message is not null. then we don't need '?'

What do you suggest?

Reply:
Snackbar type is a nullable receiver, hence I get an error if I don't add '?'